src/main/java/com/icthh/xm/tmf/ms/resource/lep/XmMsLepProcessingApplicationListener.java

```diff
@@ -1,23 +1,19 @@
 package com.icthh.xm.tmf.ms.resource.lep;
 
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_KEY_COMMONS;
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_KEY_SERVICES;
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_KEY_TEMPLATES;
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_SUB_KEY_PERMISSION_SERVICE;
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_SUB_KEY_SERVICE_TENANT_CONFIG_SERICE;
-import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.BINDING_SUB_KEY_TEMPLATE_REST;
-
 import com.icthh.xm.commons.config.client.service.TenantConfigService;
 import com.icthh.xm.commons.lep.commons.CommonsExecutor;
 import com.icthh.xm.commons.lep.commons.CommonsService;
 import com.icthh.xm.commons.lep.spring.SpringLepProcessingApplicationListener;
 import com.icthh.xm.commons.permission.service.PermissionCheckService;
 import com.icthh.xm.lep.api.ScopedContext;
-import java.util.HashMap;
-import java.util.Map;
 import lombok.RequiredArgsConstructor;
 import org.springframework.web.client.RestTemplate;
 
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.icthh.xm.tmf.ms.resource.lep.LepXmAccountMsConstants.*;
+
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version